static/templates/pod.html

```diff
@@ -0,0 +1,12 @@
+<div class="panel panel-default">
```

Comment:
I'm ok with leaving them where they are for now if you want to wrap this up. I think tho you have to change {{ to [[ for this particular template to work.